### PR TITLE
Solved issue #108 - Multiple MISSPELLINGS

### PR DESCRIPTION
**Pull request by fykaa**
1) Made grammatical corrections and improved vocabulary for the main content in index.html 
2) Updated indentations and silly errors in abbreviations.
3) Upgraded content design for Studyroom.html
4) Added professional tone to the academics.html
5) Polished the educational content on website development notes.
6) Cleared errors on the interview-prep.html.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -178,21 +178,11 @@ <h1>We Are <span id="aboutHeading">Acadmica</span></h1>
...
-            Worry not as you have come to the right place. We are providing
-            notes, study materials, coding questions for placements, compiled at
-            one place. <br />
+            Worry not as you have come to the right place. We are providing notes, study materials, coding questions for placements, all compiled at one place. <br />
             <br />
-            We go through the internet and find the best resources for you and
-            serve you at one place. Also if a student feels that his notes or
-            study materials can help others, he or she can share material with
-            us, in return you will be getting fame on social media handles.
+            We go through the internet and find the best resources for you and serve you at one place. Also, if a student believes that his or her notes/study materials can benefit others, he or she can submit the materials to us, in exchange for fame on social media handles.
```
Comment (narayan954) [inline index.html:185]: Just as a suggestion , would saying "and get featured on our social media handles" be better than "in exchange for fame on social media handles." ?

File: src/pages/interviewPrep/main.html [hunk 2]
```diff
@@ -109,7 +109,7 @@ <h1>now<span class="exclaimation">!</span></h1>
     <!-- Explore tracks section -->
     <section class="tracks-section">
       <h1 class="text-center">The first question</h1>
-      <h1 class="text-center"><span>which language</span>? to start with.</h1>
+      <h1 class="text-center"><span>which language</span> to start with?</h1>
```
Comment (narayan954) [inline src/pages/interviewPrep/main.html:112]: Would capital "Which" be better?

**Review by narayan954 — CHANGES_REQUESTED** [review]
Great work! , please check my comments and see to it if they need action.

File: index.html (hunk 1 above)
Comment (fykaa) [reply index.html]: Hi @narayan954 Thanks a lot for addressing. Updated it!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Inefficient

File: src/pages/interviewPrep/main.html (hunk 2 above)
Comment (fykaa) [reply src/pages/interviewPrep/main.html]: Hi @narayan954 Thanks a lot for addressing. Updated it!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Comment by fykaa** [comment]
Hi @narayan954 Thanks a lot for addressing. Read your comments and made the required changes! You can recheck it)

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by narayan954 — APPROVED** [review]
great , thanks for contributing!

**Pull request merged**